### DISCUSSION TITLE
Stock question about pyboards V1.1 and D series

### DISCUSSION THREAD

**Discussion by dvanderelst** [micropython/micropython#15198] [STM32 / Pyboard]
I really like the pyboard V1.1 and especially the D-series. I've bought a batch of these boards in the past and these are just some awesome boards. However, the D-series seems permanently out of stock (See https://store.micropython.org). Does anybody know whether these boards are still fabricated (and therefore likely to become available again). If not, does anyone know why production ceased? I've tried sending an email to `contact at micropython.org`. But I never got a reply.

**Comment by peterhinch** [comment]
I believe the problem is with unavailability of one particular chip. You could try contact@georgerobotics.co.uk for a definitive answer.
